### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by Corey-Grainger**
Hi @Nathan-Dachille could you please review my code for the week 09 Prac.  In particular could you please check that I am correctly using the Super() method.  😄 Thanks for your time.

### CODE REVIEW

File: prac_09/band.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""CP1404 Week 09 Prac
+Band Class
+Estimated: 20 minutes
+Commenced 1:34pm
+Completed 2:03pm"""
+
+class Band():
```
Comment (Nathan-Dachille) [inline prac_09/band.py:8]: Great variable naming and formatting just remember to add docstrings for classes and methods.

File: prac_09/taxi.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    def start_fare(self):
+        """Begin a new fare."""
+        self.current_fare_distance = 0
+
+    def drive(self, distance):
+        """Drive like parent Car but calculate fare distance as well."""
+        distance_driven = super().drive(distance)
+        self.current_fare_distance += distance_driven
+        return distance_driven
```
Comment (Nathan-Dachille) [inline prac_09/taxi.py]: Remember to leave a empty newline at the end of a program.

File: prac_09/silver_service_taxi.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        super().__init__(**kwargs)
+        self.fanciness = fanciness
+        self.price_per_km = Taxi.price_per_km * fanciness
+
+    def __str__(self):
+        return f"{super().__str__()} plus flagfall of ${self.flagfall:.2f}"
+
+    def get_fare(self):
+        return super().get_fare() + self.flagfall
+
```
Comment (Nathan-Dachille) [inline prac_09/silver_service_taxi.py]: There are too many newlines at the end of the file (3), There should be one.

**Review by Nathan-Dachille — APPROVED** [review]
Great Job!

I had to look really hard to find much to comment on as your solutions are very well formatted with great variable naming. The only thing that I have picked up on are a couple of missing docstrings as well as some slight formatting problems that can be fixed using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>L</kbd> in PyCharm. Other then that your commit messages are well worded and frequent.

Have a great day, Nathan

**Comment by Corey-Grainger** [comment]
Thanks for the review Nathan.  I appreciate you doing it so fast so I can fix those errors.  I really need that reminder to hit the auto format sometimes too!  🤦‍♂️  Cheers

**Pull request merged**